gdx-backend-jtransc/src/com/jtransc/media/limelibgdx/LimeInput.java

```diff
@@ -327,9 +327,12 @@ public int getDeltaY() {
 	@Override
 	public int getDeltaY(int i) {
 		if (lockMouse) {
-			return pointers.get(i) == null ? 0 : (int) pointers.get(i).getDeltaY();
+			for (Map.Entry<Integer, Pointer> entry : pointers.entrySet()) {
```

Comment:
Not sure how much garbage will this generate. If you have problems with GC/pauses, maybe we can create two fixed size arrays (something like 16 elements should be enough) one for ids and other for values.

I wouldn't do this unless this is necessary. This is just a comment for a possible future optimization.

Also if you find it useful I could add a method to get the number of allocations/bytes allocated of the application from java objects somewhere in jtransc. That would allow you to create a delta of objects allocateds per frame.